mysql-test/main/load_timezones_with_alter_algorithm_inplace.test

```diff
@@ -0,0 +1,36 @@
+set global alter_algorithm=INPLACE;
+
+# Because loading timezones alters the mysql tables,
+# this test will leave mysql in a different state than when it started.
+# Furthermore, checksums on the various mysql.timezone_x tables will fail. 
```

Comment:
Good docs.

Although there is a couple of instances of `set global x=DEFAULT` as a restore mechanism (most are `session` based restores), its generally better to use user variables (a different one from the maradb-tz-info-to-sql).

Good practice to list a MDEV.